src/components/calcite-value-list-item/calcite-value-list-item.scss

```diff
@@ -18,3 +19,24 @@ calcite-pick-list-item[selected] {
   z-index: 1;
   @include borderShadowActive();
 }
+
+.handle {
+  display: flex;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  margin-bottom: 1px;
+  padding: var(--calcite-app-cap-spacing-three-quarters) var(--calcite-app-side-spacing-half);
+  background-color: var(--calcite-app-background);
+  color: var(--calcite-app-foreground-subtle);
+  line-height: 0;
+  cursor: move;
+  @include borderShadow();
+  &--activated {
+    background-color: black;
+    color: white;
+  }
```

Comment:
@asangma please update these styles. They are placeholder.

Reply:
Will do. I think we should use an explicit name e.g. `--dragging`.

Reply:
Name can change. Just bear in mind this is the state for after a use has focused the handle, and pressed space to activate keyboard sorting. The actual dragging isn't animated when using the keyboard, the positions jump to their new spots.

Reply:
@pr3tori4n Ah...nvmd. I see that it's related to the handle.